classlib/src/main/java/org/teavm/classlib/java/io/TPrintStream.java

```diff
@@ -31,7 +33,7 @@
 import org.teavm.classlib.java.nio.charset.impl.TUTF8Charset;
 import org.teavm.classlib.java.util.TFormatter;
 
-public class TPrintStream extends TFilterOutputStream {
+public class TPrintStream extends TFilterOutputStream implements TAppendable {
```

Comment:
Take a look to the bottom of the file: there's a wrapper that create `Appendable` out of this `PrintStream`. I think that now `getAppendable` can be removed and formatters can take directly `this` reference

Reply:
I made TPrintStream implement TAppendable, and not java.lang.Appendable interface. So we can't get rid of getAppendable() unless TPrintStream implements Appendable.

I didn't find guidelines for design of classlib classes, so intuitively decided that class with the name starting with T should implement T interfaces.

If you are OK to make TPrintStream implement java.lang.Appendable instead of TAppendable I can alter the fix.

Reply:
TeaVM does following:

1. As a class required and it's in a package starting with `java.`, adds `org.teavm.classlib.` prefix to the package name and `T` prefix to the class name and takes this class, then renames it back to initial name.
2. After any class is parsed, all occurrences of classes (in method parameters, invocation instructions, etc) are renamed if they are in package `org.teavm.classlib.java.` using previous pattern.

So it does not matter if you implement `Appendable` or `TAppendable`.

Reply:
Updated the fix as agreed. TPrintStream looks cleaner now